wagtail/wagtailadmin/static_src/wagtailadmin/js/userbar.js

```diff
@@ -1,18 +1,33 @@
 'use strict';
 
 document.addEventListener('DOMContentLoaded', function() {
-    var body = document.querySelectorAll('body')[0];
-    var nav = document.querySelectorAll('nav')[0];
-    var className = 'ready';
-    var hasPostMessage = window.postMessage;
+    var trigger = document.querySelectorAll('[data-wagtail-userbar-trigger]')[0];
```

Comment:
If you're only looking for the first object to match, you can use `.querySelector` instead.